src/commands/mod.rs

```diff
@@ -1099,10 +1104,11 @@ pub struct ListCoinsResult {
     pub coins: Vec<ListCoinsEntry>,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Instead of removing them here, add them to `CreateSpendWarning`.

Reply:
Thanks, I've done that now. Something before made me think it wasn't working, but it does :relieved: